client/community/action-creators/dns-control/async-add-hosted-zone.js

```diff
@@ -0,0 +1,24 @@
+import * as t from '../../action-types'
+import { createAction } from '../create-action'
+import * as CommunitySelectors from '../../selectors'
+// Dependency module
```

Comment:
Now, we can remove those comments that specify the "responsibility" of the dependency. Those comments was useful on the past, when we did not have the import based on alias.